docs/javascript/index.yml

```
@@ -37,14 +37,25 @@ landingContent:
   # Card (optional)
   - title: Create JavaScript and TypeScript apps
     linkLists:
+      - linkListType: quickstart
+          - text: Create an Angular App
```

```suggestion
      - linkListType: quickstart
         links:
          - text: Create an Angular App
```

we'll need to move this to the private repo so we can check the build on staging before we push to master.... but I believe the above line should be added.